package.xml

```diff
@@ -0,0 +1,32 @@
+<?xml version="1.0"?>
+<package>
+  <name>h264_image_transport</name>
+  <version>0.0.0</version>
+  <description>The h264_image_transport package</description>
+
+  <maintainer email="adl@ut.ee"> University of Tartu Autonomous Driving Lab</maintainer>
```

Comment:
Add yourself as `<author>`.

Reply:
done

Reply:
This doesn't seem to be done - forgot to commit?

Reply:
indeed, forgot to add the file before comit.

Reply:
done